myning/objects/combat_game.py

```diff
@@ -143,6 +143,9 @@ def __str__(self) -> str:
         s += "\n╚" + "═" * (8 * len(self.block_keys) + 5) + "╝\n\n"
 
         s += self._score_str
+
+        s += term.snow4("\n\nIf you don't play the game, it won't penalize you.\n")
```

Comment:
suggest same as the other

Reply:
You're not gonna make an easy commit for me? :D